platform/mwm_version.cpp

```diff
@@ -13,6 +13,13 @@
 
 namespace version
 {
+int64_t constexpr kASingleMwmVersionForTesting1 = 991215;
```

Comment:
It's ok to define integral constexpr's in a header file, no need in extern here.